site/docs/reference/authentication.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 1
+---
+
+# Authenticating Flow
```

Comment:
This page seems to confuse authentication with authorization in a few places. Authentication is the act of verifying the _identity_ of a user. Authorization is deciding what actions a given user is allowed to take. I think the tldr is you can just replace "Authentication" with "Authorization" throughout this page. I'd expect an authentication page to answer the question of "how can users sign in?", which might also be a useful (and brief) page to have at some point.

Apart from that, and some other really minor nits, I think this page is really good and well written.